gui/index.go

```diff
@@ -43,6 +44,12 @@ func (ui *GUI) renderIndex(w http.ResponseWriter, r *http.Request, modalError st
 		address = ui.cfg.Domain
 	}
 
+	_, minerPort, err := net.SplitHostPort(ui.cfg.MinerListen)
+	if err != nil {
+		log.Errorf("Failed to parse port from miner listening address %q",
```

Comment:
Error message is capitalized. Any reason why the error itself is not included in the output?